src/main/java/com/bitalino/comm/BITalinoSocket.java

```diff
@@ -72,7 +74,12 @@ public BITalinoSocket(final DataInputStream is, final OutputStream os) {
             buffer[0] = bTemp[0];
             f = BITalinoFrameDecoder.decode(buffer, analogChannels, totalBytes);
           }
+        } else if (f.getSequence() != (prevSeq+1)%16){
+          System.out.println("Sequence out of order.")
```

Comment:
This is a library and as such we shouldn't print stuff but rather throw an exception.